qubes.SaltLinuxVM

```diff
@@ -42,6 +42,10 @@ mkdir -p /etc/salt
 if [ -f /srv/master ]; then
   mv /srv/master /etc/salt/
 fi
+if [ -d /srv/_extmods ]; then
+    mkdir -p /var/cache/salt/master
+    mv /srv/_extmods /var/cache/salt/master/extmods
```

Comment:
Not really a fan of doing it this way, would feel better having `/srv`, `/extmods`, `master` as separate entries in the tempdir. Just changing the hierarchy would break on one-sided package upgrades though, so at least for a grace period, we would have to copy `srv` into `<tempdir>` and `<tempdir>/srv` anyways. Leaving it this way for now.

Reply:
The above will misbehave if `/var/cache/salt/master/extmods` already exists (it will put _extmods inside).